.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: php
+
+git:
+  depth: 1
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+sudo: false
+
+env:
+  global:
+    - COMPOSER_ALLOW_XDEBUG=0
+  matrix:
+    - COMPOSER_COMMAND=install
+    - COMPOSER_COMMAND=update
```

Comment:
Why do we need both `install` and `update`? Checking the installed dependencies should suffice.

Reply:
I was not sure if installing the same lock file in the different PHP versions is enough and thought maybe the security checker does something different in a new version.

But if you don’t like it, I think we can remove it. Should I?

Reply:
No, you are right I think. The dependencies will be different depending on the PHP version.

Reply:
But we only need to check `composer install` with the lowest PHP version, because the lock file is built against PHP 5.6 (Contao 4.4) or PHP 7.1 (Contao 4.5).

Reply:
OK, should I change it so that we run `composer install` for PHP 5.6 and `composer update` for all other PHP versions?

Reply:
Yes please.

Reply:
Done.